### ISSUE TITLE
Cut distribution-spec release with reference types

### ISSUE THREAD

**Issue by lachie83** [opencontainers/distribution-spec#347]
Now that the reference types working group proposal has been merged into main, I would like to start the discussion on getting [image-spec](https://github.com/opencontainers/image-spec/issues/952) and corresponding distribution-spec releases cut. The reference types work was merged via the following PRs:

* https://github.com/opencontainers/image-spec/issues/940
* https://github.com/opencontainers/distribution-spec/pull/335

There is also some cleanup work covered in the following issue - https://github.com/opencontainers/distribution-spec/issues/337

In order to ease the decision of when to cut a release, I thought creating a checklist of items that must happen prior to release (release gating criteria if you will) might be a good way to make progress towards that goal. I have come up with the following list and am happy to add more items that might have been missed.

- [ ] Closure of outstanding tasks
- [ ] Conformance tests
- [ ] Sufficient Documentation
- [ ] OPTIONAL Implementation endorsements (potential list of registry providers that have agreed to implement the changes)
- [ ] Versioning agreement (if/how these changes to the image-spec constitute an API version change)

**Comment by sudo-bmitch** [comment]
I'm closing this one out since we finally had a 1.1.0 release earlier this year. Thanks for all your help getting us there @lachie83!
